TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/netty/BacksideHttpWatcherHandler.java

```diff
@@ -26,6 +27,11 @@ public BacksideHttpWatcherHandler(AggregatedRawResponse.Builder aggregatedRawRes
     protected void channelRead0(ChannelHandlerContext ctx, FullHttpResponse msg) throws Exception {
         doneReadingRequest = true;
         triggerResponseCallbackAndRemoveCallback();
+//      TODO: Add requestId to this metrics log entry
+        metricsLogger.atSuccess()
+                .addKeyValue("channelId", ctx.channel().id().asLongText())
+                .addKeyValue("httpStatus", msg.status().toString())
```

Comment:
Cool!  I forgot that we ARE parsing the HTTP responses right now (still).

Reply:
yeah, this was very useful!